### PR TITLE
feat(files): properly format buttons, align mtime to the left and apply opacity based on file last modification

### PR DESCRIPTION
**Pull request by skjnldsv**
- Mtime is alligned left
- Mtime needs colour fade from now to 31 days old
- Buttons should not be messed with the padding as `padding-block` is used and not simple padding

image

### CODE REVIEW

File: apps/files/src/components/FileEntry.vue [hunk 1]
```diff
@@ -389,6 +390,24 @@ export default Vue.extend({
 			}
 			return this.t('files_trashbin', 'A long time ago')
 		},
+		mtimeOpacity() {
+			// Whatever theme is active, the contrast will pass WCAG AA
```
Comment (artonge) [inline apps/files/src/components/FileEntry.vue:394]: So I assume that this accessible.
@JuliaKirschenheuter wasn't this remove from the old files view?

Comment (skjnldsv) [reply apps/files/src/components/FileEntry.vue:394]: Jan talked to me about it recently and said it was missing :)
but yes, it does pass WCAG AA (not AAA, which is ok for secondary text)
image

**Review by artonge — APPROVED** [review]
Let's maybe wait for @JuliaKirschenheuter input

**Review by szaimen — APPROVED** [review]

File: apps/files/src/components/FileEntry.vue (hunk 1 above)
Comment (skjnldsv) [reply apps/files/src/components/FileEntry.vue:394]: cc @jancborchardt

**Pull request merged**